src/parsers/index.js

```diff
@@ -2,6 +2,7 @@ const testng = require('./testng');
 const junit = require('./junit');
 const xunit = require('./xunit');
 const mocha = require('./mocha');
+const cucumberjs = require('./cucumberjs');
```

Comment:
Can we rename the file to just **cucumber**

Reply:
Updated